features/core/login.feature

```diff
@@ -5,71 +5,160 @@ Scenario: Logging in to evan.network using angular
   Given I log in to evan.network using angular
   Then I can see the angular dashboard
 
+
 @tag:noLogout
 Scenario: Logout from of evan.network using angular
 
   Given I log in to evan.network using angular with default
   When I log out from angular
   Then I am no longer logged in to angular
 
+
+@CORE-461
 Scenario: Logging in to evan.network using vue
 
   Given I log in to evan.network using vue
   Then I want to see a text including 'What would you like to start with?'
 
+
+@tag:noLogout
+@CORE-461
+Scenario: Logout from of evan.network using vue
+
+  Given I log in to evan.network using vue with default
+  When I log out from vue
+  Then I am no longer logged in to vue
+
+
+@tag:noLogout
+@CORE-461
+Scenario: Entering a correct mnemonic
+
+  Given I go to the evan.network startpage
+  Then I want to see a text including 'Please select an account type.'
+  When I click on link to "sign-in"
+  Then I want to see a text including 'Recovery Key'
+  When I set Input field with id "mnemonicInput0" to "define"
+    And I set Input field with id "mnemonicInput1" to "sunset"
+    And I set Input field with id "mnemonicInput2" to "extra"
+    And I set Input field with id "mnemonicInput3" to "swear"
+    And I set Input field with id "mnemonicInput4" to "runway"
+    And I set Input field with id "mnemonicInput5" to "such"
+    And I set Input field with id "mnemonicInput6" to "sleep"
+    And I set Input field with id "mnemonicInput7" to "milk"
+    And I set Input field with id "mnemonicInput8" to "alley"
+    And I set Input field with id "mnemonicInput9" to "text"
+    And I set Input field with id "mnemonicInput10" to "lemon"
+    And I set Input field with id "mnemonicInput11" to "stadium"
+    Then the button "Next" should be "enabled"
```

Comment:
I think the "Thens" sould be on the same indentation level as the "Whens"

Reply:
Used indent makes more sense, when seen as chained operations like:
```
const blubb = []
  .filter()
  .map()
```

Reducing indent of the last line would make it look a bit weird.
```
const blubb = []
  .filter()
.map()
```

But be honest, I couldn't find guidelines for indents. Most of the examples are short and not indented at all... :/

@karladler

Reply:
maybe take a look here: https://cucumber.io/docs/guides/10-minute-tutorial/#write-a-scenario